react_frontend/src/components/Home.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react'
 import '../assets/NavBar.css'
+import SearchBar from './SearchBar'
+import SearchButton from './SearchButton'
 
 export default function Home() {
     return (
         <div>
-            <p>hi this is the home page</p>
+        <div className="container">
+        {/*<Table characterData={characters} removeCharacter={this.removeCharacter} />
```

Comment:
LGTM might want to remove the comments before push though

Reply:
I will try to keep comments organized if there are any for future pull requests; I originally left it in thinking some of it might come in handy for tasks I might do later on.